vcbuild.bat

```diff
@@ -252,10 +254,24 @@ goto build-doc
 
 :project-gen
 @rem Skip project generation if requested.
-if defined noprojgen goto msbuild
+SETLOCAL EnableDelayedExpansion
+if defined noprojgen goto skip-configure
+if defined forceprojgen goto run-configure
+if not exist node.sln goto run-configure
+if not exist used_configure_flags goto run-configure
+set /p prev_configure_flags=<used_configure_flags
+if NOT !prev_configure_flags! == !configure_flags! goto run-configure
+
+:skip-configure
+ENDLOCAL
+echo Reusing solution generated with %prev_configure_flags%
+goto msbuild
 
+:run-configure
+ENDLOCAL
 @rem Generate the VS project.
 echo configure %configure_flags%
+echo %configure_flags%> used_configure_flags
```

Comment:
Can we rename the tmp file `.used_configure_flags`. I know this is not *nix, but git will exclude it by default, and Windows will at least group it together with the rest of the settings/temporary files.